ladybug_comfort/et.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""Utility functions to calculate Effective Temperature"""
+
+
+def effective_temperature_conditions(tre, te):
+    """ Inputs:
+        * tre: Radiant effective temperature
+        * te: Thermal emission
+        Outputs:
+        * tre: Radiant effective temperature
+        * te: Effective temperature
+    """
+    if tre < 1:
+        effectTE = -4
+    elif tre >= 1 and te < 9:
+        effectTE = -3
+    elif tre >= 9 and te < 17:
+        effectTE = -2
+    elif tre >= 17 and te < 21:
+        effectTE = -1
+    elif tre >= 21 and te < 23:
+        effectTE = 0
+    elif tre >= 23 and te < 27:
+        effectTE = 1
+    elif tre >= 27:
+        effectTE = 2
+
+    return tre, effectTE
+
+
+def effective_temperature(ta, ws, rh, sr, ac):
+    """ Inputs: 
+            * ta: Air Temperature 
+            * ws: Wind Speed
+            * rh: Relative Humidity
+            * sr: Solar Radiation
```

Comment:
Also, what are the units of solar radiation here? And the units of all of these inputs for that matter?